packages/flutter/lib/src/widgets/scroll_notification.dart

```diff
@@ -253,6 +255,24 @@ class ScrollEndNotification extends ScrollNotification {
   }
 }
 
+/// A notification that a [Scrollable] widget has cancelled scrolling.
+///
+/// Cancelling scroll means [Scrollable] widget restores to the state before
```

Comment:
So, if I start scrolling downward and then change direction to scroll back up to the same position that would be considered a cancelled scroll? That seems odd...